pilot/pkg/xds/testdata/benchmarks/authorizationpolicy.yaml

```
@@ -0,0 +1,36 @@
+# Set up a Service associated with our proxy, which will run as 1.1.1.1 IP
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: proxy-service-instance
+spec:
+  hosts:
+    - example.com
+  ports:
+    - number: 80
+      name: http
+      protocol: HTTP
+    - number: 443
+      name: https
+      protocol: HTTPS
+  resolution: STATIC
+  endpoints:
+    - address: 1.1.1.1
+      labels:
+        istio.io/benchmark: "true"
+---
+  {{- range $i := until .Services }}
+apiVersion: security.istio.io/v1beta1
+kind: AuthorizationPolicy
+metadata:
+  name: authn-{{$i}}
+spec:
+  action: DENY
+  rules:
+    - from:
+        - source:
+            namespaces: ["default"]
+      to:
+        - operation:
+            methods: ["POST"]
+  {{- end }}
```

```suggestion
---
  {{- end }}
```

otherwise there is no delimiter and it only reads one of them. same with the other file